test/core/store/stores/test_directory.py

```diff
@@ -162,23 +162,23 @@ def test_get_data_ids(self):
         )
         self.assertEqual(
             {
-                ('cube-1-250-250.zarr', None),
-                ('cube-5-100-200.zarr', None),
-                ('cube.nc', None),
+                ('cube-1-250-250.zarr', {}),
+                ('cube-5-100-200.zarr', {}),
+                ('cube.nc', {}),
             },
-            set(self.store.get_data_ids(include_titles=False))
+            set(self.store.get_data_ids(include_attrs=["title"]))
```

Comment:
As this is not the expected behaviour, I recommend to comment this part of the test out as long as the stores do not return titles (or any other attributes).

Reply:
This _is definitely_ the expected behaviour for _this_ implementation. I agree, adding a comment would be good. Once we intent to change behaviour, we do this by a test-driven aypproach and _change the test accordingly first_.